Error printing Doubles in object property

When yo have an object with a reference to a Int it can be printing in REPL ok:
```Xtend
object pepita {
	var energia = 100
}

>>> pepita
pepita[energia=100]
```

But when it reference to a Double the printing shows only the name:
```Xtend
object pepita {
	var energia = 100.5
}

>>> pepita
pepita
```

If a test it in Wollok Development I get:
```java
java.lang.AssertionError: Error while calling native method 'WCollection.fold(WollokObject, WollokObject) with parameters: [edad=23], aClosure(acc, e)
wollok.lang.Exception: Error while calling native method 'WCollection.fold(WollokObject, WollokObject) with parameters: [edad=23], aClosure(acc, e)
	at wollok.lang.List.fold(initialValue,closure) [/wollok.wlk:568]
	at wollok.lang.Collection.map(closure) [/wollok.wlk:373]
	at wollok.lang.Object.internalToSmartString(alreadyShown) [/wollok.wlk:143]
	at wollok.lang.Object.internalToSmartString(alreadyShown) [/wollok.wlk:142]
	at wollok.lang.Object.toSmartString(alreadyShown) [/wollok.wlk:133]
	at wollok.lang.Object.toString() [/wollok.wlk:125]
	at  [__synthetic0.wpgm]
Caused by: wollok.lang.Exception: Error while calling native method 'Closure.doApply(WollokObject[]) with parameters: [Lorg.uqbar.project.wollok.interpreter.core.WollokObject;@38b5f25
	at wollok.lang.Closure.apply(parameters) [/wollok.wlk:963]
	at wollok.lang.Collection.map(closure) [/wollok.wlk:374]
	at wollok.lang.List.fold(initialValue,closure) [/wollok.wlk:568]
	at wollok.lang.Collection.map(closure) [/wollok.wlk:373]
	at wollok.lang.Object.internalToSmartString(alreadyShown) [/wollok.wlk:143]
	at wollok.lang.Object.internalToSmartString(alreadyShown) [/wollok.wlk:142]
	at wollok.lang.Object.toSmartString(alreadyShown) [/wollok.wlk:133]
	at wollok.lang.Object.toString() [/wollok.wlk:125]
	at  [__synthetic0.wpgm]
**Caused by: wollok.lang.MessageNotUnderstoodException: 1.7 (WDouble) does not understand stringValue()**
	at wollok.lang.Double.stringValue() [/wollok.wlk:791]
	at wollok.lang.Number.internalToSmartString(alreadyShown) [/wollok.wlk:682]
	at wollok.lang.Object.toSmartString(alreadyShown) [/wollok.wlk:133]
	at wollok.mirror.InstanceVariableMirror.valueToSmartString(alreadyShown) [/wollok.wlk:1223]
	at wollok.lang.Object.internalToSmartString(alreadyShown) [/wollok.wlk:144]
	at wollok.lang.Object.internalToSmartString(alreadyShown) [/wollok.wlk:144]
	at wollok.lang.Closure.apply(parameters) [/wollok.wlk:963]
	at wollok.lang.Collection.map(closure) [/wollok.wlk:374]
	at wollok.lang.List.fold(initialValue,closure) [/wollok.wlk:568]
	at wollok.lang.Collection.map(closure) [/wollok.wlk:373]
	at wollok.lang.Object.internalToSmartString(alreadyShown) [/wollok.wlk:143]
	at wollok.lang.Object.internalToSmartString(alreadyShown) [/wollok.wlk:142]
	at wollok.lang.Object.toSmartString(alreadyShown) [/wollok.wlk:133]
	at wollok.lang.Object.toString() [/wollok.wlk:125]
	at  [__synthetic0.wpgm]
	at org.junit.Assert.fail(Assert.java:88)
	at org.uqbar.project.wollok.tests.interpreter.AbstractWollokInterpreterTestCase.lambda$7(AbstractWollokInterpreterTestCase.java:185)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at org.uqbar.project.wollok.tests.interpreter.AbstractWollokInterpreterTestCase.lambda$5(AbstractWollokInterpreterTestCase.java:192)
	at org.eclipse.xtext.xbase.lib.ObjectExtensions.operator_doubleArrow(ObjectExtensions.java:139)
	at org.uqbar.project.wollok.tests.interpreter.AbstractWollokInterpreterTestCase.interpret(AbstractWollokInterpreterTestCase.java:194)
	at org.uqbar.project.wollok.tests.interpreter.AbstractWollokInterpreterTestCase.interpret(AbstractWollokInterpreterTestCase.java:150)
	at org.uqbar.project.wollok.tests.interpreter.AbstractWollokInterpreterTestCase.interpret(AbstractWollokInterpreterTestCase.java:146)
	at org.uqbar.project.wollok.tests.interpreter.AbstractWollokInterpreterTestCase.interpretPropagatingErrors(AbstractWollokInterpreterTestCase.java:114)
	at org.uqbar.project.wollok.tests.interpreter.AbstractWollokInterpreterTestCase.interpretPropagatingErrors(AbstractWollokInterpreterTestCase.java:90)
	at org.uqbar.project.wollok.tests.sdk.ObjectTest.defaultToString(ObjectTest.java:73)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.eclipse.xtext.junit4.XtextRunner$1.evaluate(XtextRunner.java:49)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
```

--- a/org.uqbar.project.wollok.lib/src/wollok/lang/WInteger.xtend
+++ b/org.uqbar.project.wollok.lib/src/wollok/lang/WInteger.xtend
@@ -17,8 +17,6 @@ class WInteger extends WNumber<Integer> implements Comparable<WInteger> {
 	}
 	
 	def abs() { Math.abs(wrapped).asWollokObject }
-	
-	def stringValue() { wrapped.toString }
 
 	@NativeMessage("+")
 	def plus(WollokObject other) { operate(other) [ doPlus(it) ] }

--- a/org.uqbar.project.wollok.lib/src/wollok/lang/WNumber.xtend
+++ b/org.uqbar.project.wollok.lib/src/wollok/lang/WNumber.xtend
@@ -17,7 +17,7 @@ abstract class WNumber<T extends Number> extends AbstractJavaWrapper<T> {
 
 	new(WollokObject obj, WollokInterpreter interpreter) {
 		super(obj, interpreter)
-	}
+	}	
 	
 	def max(WNumber<?> other) { doMax(this, other).asWollokObject }
 
@@ -40,9 +40,11 @@ abstract class WNumber<T extends Number> extends AbstractJavaWrapper<T> {
 	// ** Basics
 	// ********************************************************************************************	
 	
-	def doubleValue() { wrapped.doubleValue() }
+	def doubleValue() { wrapped.doubleValue }
 
-	override toString() { wrapped.toString() }
+	def stringValue() { wrapped.toString }
+	
+	override toString() { wrapped.toString }
 	
 	override equals(Object obj) {
 		this.class.isInstance(obj) && wrapped == (obj as WNumber).wrapped 

--- a/org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/sdk/ObjectTest.xtend
+++ b/org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/sdk/ObjectTest.xtend
@@ -39,6 +39,20 @@ class ObjectTest extends AbstractWollokInterpreterTestCase {
 			}
 		'''.interpretPropagatingErrors
 	}
+	
+	@Test
+	def void toStringWithDoubleField() {
+		'''
+			object persona {
+				var edad = 23
+				var altura = 1.7
+			}
+			
+			program p {
+				assert.equals("persona[edad=23, altura=1.7]", persona.toString())
+			}
+		'''.interpretPropagatingErrors
+	}
 
 	@Test
 	def void instanceVariableFor() {

